.buildkite/test-pipeline.yaml

```diff
@@ -156,6 +156,7 @@ steps:
   - pytest -v -s distributed/test_utils.py
   - pytest -v -s compile/test_basic_correctness.py
   - pytest -v -s distributed/test_pynccl.py
+  - pytest -v -s distributed/test_events.py
```

Comment:
Could you add this file to the `source_file_dependencies` above with `test_utils` and `test_pynccl`?